pyproject.toml

```diff
@@ -1,57 +1,135 @@
-[tool.poetry]
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
 name = "data-safe-haven"
-version = "4.0.0"
+dynamic = ["version"]
 description = "An open-source framework for creating secure environments to analyse sensitive data."
-authors = ["Data Safe Haven development team <safehavendevs@turing.ac.uk>"]
+authors = [
+  { name = "Data Safe Haven development team", email = "safehavendevs@turing.ac.uk" },
+]
+requires-python = ">=3.10"
+license = "BSD-3-Clause"
+dependencies = [
+  "azure-core~=1.26.0",
```

Comment:
Is `~=` equivalent to the `^=` that poetry uses?

Reply:
Yes. Even better, it is the [PEP440](https://peps.python.org/pep-0440/#version-specifiers) and [PEP508](https://peps.python.org/pep-0508/) way of specifying versions/requirements.